lib/_kernel.scm

```diff
@@ -4279,6 +4279,12 @@ end-of-code
             scheme-object   ;; device
    "___os_device_stream_open_process"))
 
+(define-prim ##os-device-open-raw
```

Comment:
This is not portable across OSes. How is this going to be used? How does the program get to the file descriptor?